website/screens/components/select/code/SelectCodePage.tsx

```diff
@@ -253,63 +244,46 @@ const sections = [
         content: (
           <>
             <DxcText as="p">
-              The state of the component is managed internally by the select.
-              Here is an example of how to use an uncontrolled select the submit
-              event.
+              Alternatively you can let the component manage its state
+              internally. The example below shows how to handle the value of an
+              uncontrolled select in the submit event.
             </DxcText>
-            <Example example={uncontrolled} defaultIsVisible />
-          </>
-        ),
-      },
-      {
-        title: "Searchable",
-        content: (
-          <>
             <DxcText as="p">
-              The <Code>searchable</Code> prop enables the search functionality
-              for filtering the select options.
+              The Select's value is empty by default, but an initial,
+              uncontrolled, value can be provided using the{" "}
+              <Code>defaultValue</Code> prop.
             </DxcText>
-            <Example example={searchable} defaultIsVisible />
+            <Example example={uncontrolled} defaultIsVisible />
           </>
         ),
       },
       {
-        title: "Multiple",
+        title: "Error handling",
         content: (
           <>
             <DxcText as="p">
-              The <Code>multiple</Code> prop enables the user to select more
-              than one option. If the component is also optional, it will only
-              add "(Optional)" next to the label and not another option to the
-              listbox.
+              When handling errors, we recommend initializing the{" "}
+              <Code>error</Code> prop with an empty string. This will reserve
+              space for a possible future error message and prevent unintended
+              layout changes. Also, the <Code>onBlur</Code> and{" "}
+              <Code>onChange</Code> events will send <Code>undefined</Code> when
+              there is no error, so you may need to check this too to avoid the
+              same problem.
             </DxcText>
-            <Example example={multiple} defaultIsVisible />
-          </>
-        ),
-      },
-      {
-        title: "Optional",
-        content: (
-          <>
-            <DxcText as="p">
-              You can mark a select as optional by setting to <Code>true</Code>{" "}
-              the <Code>optional</Code> prop. In this particular case, the
-              component will automatically add an extra selectable option which
-              will have the empty string as value and the placeholder as a
-              label. This only affects the single selection mode, otherwise this
-              option will not be included.
+            <DxcText>
+              Below is an example of how to treat errors using the <Code>onBlur</Code> event.
```

Comment:
Refactor this line because it doesn't have subject.

Reply:
Fixed!